tests/unit/components/sl-calendar-test.js

```diff
@@ -11,32 +13,6 @@ moduleForComponent( 'sl-calendar', 'Unit | Component | sl calendar', {
     unit: true
 });
 
-test( 'Default class name is present', function( assert ) {
```

Comment:
Change usage of new Ember.A to have the new keyword removed so it is now: Ember.A